### PR TITLE
update the super-admin-export-data.feature

### PR DESCRIPTION
**Pull request by mrpau-julius**
### Summary

 
The additional step for super admin export data features. 

### Reviewer guidance

…

### References

#5138 

----

### Contributor Checklist

PR process:

- [ ] PR has the correct target branch and milestone
- [ ] PR has 'needs review' or 'work-in-progress' label
- [ ] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

Testing:

- [ ] Contributor has fully tested the PR manually
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] Critical user journeys are covered by Gherkin stories
- [ ] Critical and brittle code paths are covered by unit tests

### Reviewer Checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.rst

### CODE REVIEW

File: integration_testing/features/super-admin/super-admin-export-data.feature [hunk 1]
```diff
@@ -7,16 +7,23 @@ Feature: Super admin exports usage data
       And the learners have had interactions with the content on the device
 
   Scenario: Export session logs
-    When I click on *Download* button under *Session logs* heading
+    When I click on "Generate log file" link under *Session logs* heading
+    Then I see generating log file loading
```
Comment (radinamatic) [inline integration_testing/features/super-admin/super-admin-export-data.feature]: Then I see the loading indicator

File: integration_testing/features/super-admin/super-admin-export-data.feature [hunk 2]
```diff
@@ -7,16 +7,23 @@ Feature: Super admin exports usage data
...
+      And the text change to "Generate a new log file"
+      And the *Download* button is enabled
```
Comment (radinamatic) [inline integration_testing/features/super-admin/super-admin-export-data.feature]: Swap the order of the lines 12 & 13.

File: integration_testing/features/super-admin/super-admin-export-data.feature [hunk 3]
```diff
@@ -7,16 +7,23 @@ Feature: Super admin exports usage data
...
     Then I see *Save as* window
-    When I enter the name and loaction for the CSV file
+    When I enter the name and location for the CSV file
       And I click the *Save* button
     Then I see the session logs file saved on my local drive
 
   Scenario: Export summary logs
-    When I click on *Download* button under *Summary logs* heading
+    When I click on "Generate log file" link under *Summary logs* heading
+    Then I see generating log file loading
```
Comment (radinamatic) [inline integration_testing/features/super-admin/super-admin-export-data.feature]: Same as line 11.

File: integration_testing/features/super-admin/super-admin-export-data.feature [hunk 4]
```diff
@@ -7,16 +7,23 @@ Feature: Super admin exports usage data
...
+      And the text change to "Generate a new log file"
+      And the *Download* button is enabled
```
Comment (radinamatic) [inline integration_testing/features/super-admin/super-admin-export-data.feature:23]: Swap the lines 23 & 24.

**Review by radinamatic — CHANGES_REQUESTED** [review]
Looking good, just a few changes needed.

**Comment by mrpau-julius** [comment]
Thank you @radinamatic for reviewing my PR. I will change that later.

**Comment by indirectlylit** [comment]
Is this ready for re-review or still a WIP?

Please set the appropriate labels

**Review by radinamatic — APPROVED** [review]
Looking good! :+1:

**Pull request merged**